prob-5/prob5.hs

```diff
@@ -0,0 +1,12 @@
+isDivisible :: Integer -> Integer -> Bool
+isDivisible _ 1 = True
+isDivisible n x
+    | n `mod` x == 0 = isDivisible n (x - 1)
+    | otherwise = False
+
+findNum :: Integer -> Integer
+findNum n
+    | isDivisible n 20 = n
+    | otherwise = findNum (n + 20)
```

Comment:
This (otherwise.. n + 20) is almost imperative style (looks like a for loop) and discouraged. See if you can use `filter` to simplify the code – you can create a list of all integers divisible by all the numbers from 1..20.